### DISCUSSION TITLE
ESPhome examples and configuration entry for Xiao based gadgets

### DISCUSSION THREAD

**Discussion by 256ericpan** [Seeed-Studio/OSHW-XIAO-Series#14] [Open for Vote]
https://esphome.io/#cookbook 
Too many details to handle to get a Xiao based device up and running in ESPhome

**Comment by limengdu** [comment]
What do you mean by Too many details? I was under the impression that configuring the pins would work, and that there wasn't much difference between the XIAO and other ESP development boards.
